assets/.well-known/manifest.webmanifest

```diff
@@ -5,13 +5,13 @@
   "version": "0.0.1",
   "icons": [
     {
-      "src": "images/smart-id-128.png",
+      "src": "../images/smart-id-128.png",
```

Comment:
Maybe just `/images/...`? [iwa-sink](https://github.com/chromeos/iwa-sink/blob/main/public/.well-known/manifest.webmanifest) does this.

Reply:
Makes sense, I'll do that.